pages/about.html

```diff
@@ -5,8 +5,37 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet">
 </head>
+
 <body>
-  
+  <section>
+    <header> 
+    <h1>Katie Diaz</h1>
+    <nav>
+      <ul class="main-nav">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about.html">About</a></li>
+      <li><a href="portfolio.html">Portfolio</a></li>
+    </ul>
+    </nav>
+  </header>
+
+    <h2 class="about-title">About me</h2>
+    <div class="about-katie">
+    <img src="images/107636937_1122230274816506_5738730028904550504_n.jpg" alt="Photo of Katie smiling">
+    <p>
+    Hello, my name is Katie and I am a full-stack 
+    software development student at Ada Developers Academy (C16). 
+    In my spare time I enjoy going on hikes, eating good food, 
+    and exploring waterfronts. 
+    </p>
+    </div>
+  </section>
+  <footer> contact me:
+    email: fakeemail@gmail.com
```

Comment:
Fun fact, [example.com](http://example.com/) is set up for things like this!

```suggestion
    email: katie@example.com
```